src/styles/global.scss

```diff
@@ -0,0 +1,54 @@
+@use "./colors.scss";
+@use "./variables.scss";
+
+@import url("https://fonts.googleapis.com/css2?family=Ubuntu:wght@300;400;500;700&display=swap");
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+  background-color: colors.$background;
+  font-family: variables.$ubuntu;
+  color: colors.$grey;
+}
+
+* {
+  font-family: variables.$ubuntu;
+  padding: 0;
+  margin: 0;
+}
+
+h1 {
+  font-size: 90px;
+  line-height: 95px;
+  text-transform: uppercase;
+}
+
+h2 {
+  font-size: 50px;
+  line-height: 60px;
+}
+
+h3 {
+  font-size: 40px;
+}
+
+h4 {
+  font-size: 30px;
+  line-height: 36px;
+}
+
+h5 {
+  font-size: 22px;
+  line-height: 34px;
+}
+
+h6 {
+  font-size: 20px;
+  line-height: 32px;
+}
+
```

Comment:
Responsive typography details missing.
Can be added in the beginning of project altogether.

Reply:
Resolved